03-rnaseq/pathway-analysis_rnaseq_03_gsva.Rmd

```diff
@@ -0,0 +1,475 @@
+---
+title: "Gene set variation analysis - RNA-seq"
+author: "CCDL for ALSF"
+date: "December 2020"
+output:  
+  html_notebook:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+**IDENTICAL TO MICROARRAY GSVA** (except microarray -> rna-seq)
+
+# Purpose of this analysis
+
+This example is one of pathway analysis module set, we recommend looking at the [pathway analysis table below](#how-to-choose-a-pathway-analysis) to help you determine which pathway analysis method is best suited for your purposes.
+
+In this example we will cover a method called Gene Set Variation Analysis (GSVA) to calculate gene set or pathway scores on a per-sample basis.
+Gene set variation analysis (GSVA) constitutes a starting point to build pathway-centric models of biology.
+Rather than contextualizing some results you have received from another analysis like DGE, GSVA is designed to provide an estimate of pathway variation for each of the samples in an experiment.
+This means that GSVA scores will depend on the samples included in the dataset when you run GSVA; if you added more samples and ran GSVA again, you would expect the scores to change [@Hanzelmann2013].
+
+GSVA determines the relative enrichment of gene sets across samples using a non-parametric approach.
+GSVA transforms a gene by sample gene expression matrix into a gene set by sample pathway enrichment matrix [@Hanzelmann-github].
+You can use these scores for other downstream analyses; in this analysis we will perform differential expression to determine what pathway is most different across our variable of interest. 
+
+⬇️ [**Jump to the analysis code**](#analysis) ⬇️
+
+**IDENTICAL TO RNA-Seq ORA** (except microarray -> rna-seq)
```

Comment:
I will just say that my preference would probably be to have this as a _comment on the pull request_, i.e., you file a PR, you "review" first to leave comments with this information, return the "review" and then request reviewers. My reasoning is two fold: 1. Zero chance of forgetting to take it out 2. I think it's would be little more noticeable as a reviewer.

Reply:
I can do comments! I like that idea.